Jenkinsfile

```diff
@@ -133,7 +133,7 @@ spec:
       limits:
         memory: "3Gi"
   - name: jakartaeetck-ci
-    image: jakartaee/cts-base:0.2
+    image: jakartaee/cts-base:0.3
```

Comment:
Could https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/docker-images/cts-base be updated with Dockerfile used to produce 0.3 + entry in README, please?

Reply:
Created https://github.com/eclipse-ee4j/jakartaee-tck/pull/824. Thanks!